transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java

```diff
@@ -132,6 +132,10 @@ private void setActiveOnOpen(boolean activeOnOpen) {
         this.activeOnOpen = activeOnOpen;
     }
 
+    boolean getActiveOnOpen() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)